frontend/TestRun/ResultsTab.svelte

```diff
@@ -6,17 +6,74 @@
     export let test_id = "";
     let results = [];
 
+    const transform_to_table = (table_data) => {
```

Comment:
should be `transformToTable` here (and `tableData`)

Reply:
python style 😆

Reply:
fixed